docs/design/mono/diagnostics-tracing.md

```diff
@@ -474,6 +515,39 @@ TraceEvent library, https://www.nuget.org/packages/Microsoft.Diagnostics.Tracing
 
 https://github.com/lateralusX/diagnostics-nettrace-samples includes a couple of custom tools (startup tracing, instrumented method execution, GC heap dump analysis) analyzing nettrace files using TraceEvent library.
 
+## Profiled AOT using `dotnet-pgo`
+
+By collecting a .nettrace file using the diagnostic server and EventPipe, it is possible to create `.mibc` files using the `dotnet-pgo` tool that can be used to tell the Mono AOT compiler which methods in an applcation should be compiled with AOT and which should be interpreted or JITed.  A typical use-case is to improve the startup performance of a mobile app, or to control the overall size of an AOTed application.
+
+A typical workflow is:
+
+1. Install the `dotnet-pgo` tool (**TODO** are there pre-built binaries?)
+2. Run the application with diagnostics configured to use the provider `Microsoft-Windows-DotNETRuntime:0x1F000080018:5` and generate a `.nettrace` file
+3. Either manually create a `.mibc` file using
+
+   ```
+   dotnet-pgo create-mibc --trace <file.nettrace>
+   ```
+
+   and pass the resulting `.mibc` file to the Mono AOT compiler MSBuild task using the `MibcProfilePath` property,
```

Comment:
Didn't we need to pass in a path to the assemblies used producing the .nettrace files to dotnet-pgo, @mdh1418?

Reply:
Yea this is incomplete. I need to update this with the actual instructions